contracts/TokenVesting.sol

```diff
@@ -20,9 +20,17 @@ contract TokenVesting is Ownable {
   event TokensClaimed(uint256 tokensClaimed);
   event VestingKilled();
 
+   /**
+   *@dev contract contructor
+   *@param _token address of erc20 token
+   *@param _receiver address entitle to claim tokens
+   *@param _startTime moment when counting time starts
+   *@param _cliff delay from _startTime after which vesting starts
+   */
   constructor(
     address _token,
     address _receiver,
+	//I believe this two variables should be replaced with one - that is _vestingStartTime equal to _startTime.add(_cliff)
```

Comment:
Thats possible, thanks for the suggestion but we want to keep both variables, please remove the comment.